### DISCUSSION TITLE
Proje Hakkında

### DISCUSSION THREAD

**Discussion by erdemklync** [python/python-docs-tr#96] [Fikirler]
Merhaba,

Öncelikle hepinizin emeklerine sağlık. Yazılım dünyasında yerelleştirme işleri gördüğümde mutlu oluyorum. :) İsmim Erdem, Mütercim-Tercümanlık lisansına sahip olsam da şu anda yazılım sektöründe çalışıyorum. Daha öncesinde bilişim teknolojileri alanında yerelleştirme projelerinde çevirmen olarak çalışıyordum. Genellikle de buna benzer referans ve yardım içerikleri çeviriyordum. Zaman bulabilirsem ben de katkıda bulunmaya çalışacağım. En azından çevirilerin revizyonlarını ve son okumalarını yapıp notlar bırakabilirim.

Naçizane, bir önerim olacak. Projenin ortak bir çeviri belleği ve terimce üzerinden ilerlemesinin daha sağlıklı olacağını düşündüm. Bu şekilde ilerleyebilirseniz daha hızlı ve tutarlı çeviriler ortaya çıkacaktır. PoEdit yerine bulut desteği olan SmartCat gibi çeviri araçları kullanabilirsiniz. Böylece yeni çeviriler yaparken daha önce yapılan çevirilerden öneriler görebilirsiniz. Ayrıca bir terimce oluşturarak terim ifadeler için farklı karşılıklar kullanılmasının önüne geçebilirsiniz.

Başarılar ve kolaylıklar diliyorum. 🙂

**Comment by egeakman** [comment]
Düşünceleriniz için teşekkürler @erdemklync 😊

> Naçizane, bir önerim olacak. Projenin ortak bir çeviri belleği ve terimce üzerinden ilerlemesinin daha sağlıklı olacağını düşündüm.

Çevirinin kendi içinde bir [sözlüğü](https://github.com/python/python-docs-tr/blob/3.11/glossary.po) olmasına rağmen çok göz önünde değil. Belki kullanımını kolaylaştıracak bir scriptle öne çıkarabiliriz. Örneğin verilen kelimenin **glossary.po** dosyasındaki karşılığını döndürebilir bu script.

> PoEdit yerine bulut desteği olan SmartCat gibi çeviri araçları kullanılabilirsiniz.

[Poedit](https://github.com/vslavik/poedit)'i kullanıyor olmamızın asıl nedeni open-source olması ve basitliğiydi, ancak Transifex veya SmartCat gibi servisler de tabii ki düşünülebilir. Biraz daha detaylandırarak (GitHub entegrasyonu, bireysel katkılar, ekip kullanımı vs.) öneride bulunabilirseniz çok seviniriz. Portekizce çeviri ekibi Transifex kullanıyor mesela belki faydalanabilirsiniz repo'larından: https://github.com/python/python-docs-pt-br.

Katkılarınızı ve fikirlerinizi her zaman bekliyoruz, iyi günler! 😊
